turbopuffer/namespace.py

```diff
@@ -185,6 +185,19 @@ def update_schema(self, schema_updates: NamespaceSchema):
         response = self.backend.make_api_request('namespaces', self.name, 'schema', method='POST', payload=request_payload)
         return parse_namespace_schema(response["content"])
 
+    def copy_from_namespace(self, other_namespace: str):
```

Comment:
this is more of a personal preference thing: `from_namespace`/`source_namespace`/`namespace_name` would look a bit nicer i think? in case one uses keyword arguments:
```
ns2.copy_from_namespace(namespace_name=ns1_name)
```
vs
```
ns2.copy_from_namespace(other_namespace=ns1_name)
```

Reply:
Went with source_namespace, much cleaner